ui/app/styles/core/forms.scss

```diff
@@ -76,12 +76,12 @@ label {
 .is-help {
   font-size: $size-small;
   margin-top: 0.25rem;
-  @each $name, $pair in $colors {
```

Comment:
same question here as above - I thought `@each` is a sass thing, not bulma specific?